### ISSUE TITLE
Linux version

### ISSUE THREAD

**Issue by jendabek** [jendabek/BlenderRenderController#9]
I have made a quick test on virtualized Ubuntu using MonoDevelop.
There are some issues but it looks like it will work :)
https://www.youtube.com/watch?v=uyncOjD_o4A

**Comment by RedRaptor93** [comment]
Very nice!

I also did a quick test on my Debian VM yesterday. But could not get it to work like it did in your vid (for instance clicking to open the recent projects also opens the file dialog) What did you change? In general?

We need to keep in mind the changes we're making to get this to work, and find a way to implement it in the project in a way it'll work on both OSes.

We could check for `Environment.OSVersion` and do changes accordingly... For instance.

**Comment by jendabek** [comment]
I have that button issue too... I was just holding the mouse button in the video :) I think this is really minor issue, the bigger one is that it not always recognize processes finish... but that test was really quick and I am sure it will be possible to fix that.
I have to continue with my 3d modeling this week so I will look at it later.
Your instance with OSVersion is of course the way to go :)

**Comment by RedRaptor93** [comment]
Getting rid of those scroll bars is easy, just need to increase the size of the Window. I'll see if I can adjust for the Ui diferences.

**Comment by jendabek** [comment]
These are not so important issues I think, much more important is to adapt checking for FFmpeg / Blender and show corresponding message / modify settings form and check actual rendering for any issues. From my quick testing it seems like it sometimes doesn't detect end of processes so it won't show Render Complete = does not continue with mixdown and joining.
This is the most important I think as it is the core functionality.
I would let those UI things at the very end.
But I am not the boss here :D Whatever you are going to tweak it will help of course :)

**Comment by RedRaptor93** [comment]
check out my 'multiPlat' branch, I made the program detect the Os and sets stuff like colors, some variables accordingly.

I did not come across that render issue, but I did notice that "total time" in my test project was incorrect (project is 16min long, BRC on linux thinks it's over 2hrs).

**Comment by RedRaptor93** [comment]
Just fixed that "Total Time" issue, `fps base` in Linux has a comma instead of a period which messed up the FPS and Time length calculations.

Can you explain more about that processes finished issue? How to replicate it?

**Comment by RedRaptor93** [comment]
UPDATE: I think I got it... 

I got a hang on one of my test runs (it rendered all frames and hung saying 3/4 processes finished) and never started mixdown. 

The mixdown process was hard-coded pointing to `BLENDER_EXE_NAME` (blender.exe), so I changed it to my new Platform specific properties.

I don't know what changes you made when you showcased in that video, maybe it was an oversight like that?

Check my [multiPlat](https://github.com/RedRaptor93/BlenderRenderController/tree/multiPlat) branch, see if that problem still happens.
